### PR TITLE
Improve error message guidance

### PR DESCRIPTION
**Pull request by Hoverbear**
##### Description

Noticed in #257 that there are others to improve too.

##### Checklist

- [ ] Formatted with `cargo fmt`
- [ ] Built with `nix build`
- [ ] Ran flake checks with `nix flake check`
- [ ] Added or updated relevant tests (leave unchecked if not applicable)
- [ ] Added or updated relevant documentation (leave unchecked if not applicable)
- [ ] Linked to related issues (leave unchecked if not applicable)

##### Validating with `install.determinate.systems`

If a maintainer has added the `upload to s3` label to this PR, it will become available for installation via `install.determinate.systems`:

```shell
curl --proto '=https' --tlsv1.2 -sSf -L https://install.determinate.systems/nix/pr/$PR_NUMBER | sh -s -- install
```

### CODE REVIEW

File: src/action/mod.rs [hunk 1]
```diff
@@ -266,11 +266,11 @@ pub enum ActionError {
         "`{0}` exists with different content than planned, consider removing it with `rm {0}`"
     )]
     Exists(std::path::PathBuf),
-    #[error("`{0}` exists with a different uid ({1}) than planned ({2}), consider removing it with `rm {0}`")]
+    #[error("`{0}` exists with a different uid ({1}) than planned ({2}), consider updating it with `chown {2} {0}`")]
```
Comment (cole-h) [inline src/action/mod.rs:269]:
```suggestion
    #[error("`{0}` exists with a different uid ({1}) than planned ({2}), consider updating it with `chown {2:o} {0}`")]
```

File: src/action/mod.rs [hunk 2]
```diff
@@ -266,11 +266,11 @@ pub enum ActionError {
...
     PathUserMismatch(std::path::PathBuf, u32, u32),
-    #[error("`{0}` exists with a different gid ({1}) than planned ({2}), consider removing it with `rm {0}`")]
+    #[error("`{0}` exists with a different gid ({1}) than planned ({2}), consider updating it with `chgrp {2} {0}`")]
```
Comment (cole-h) [inline src/action/mod.rs:271]:
```suggestion
    #[error("`{0}` exists with a different gid ({1}) than planned ({2}), consider updating it with `chgrp {2:o} {0}`")]
```

File: src/action/mod.rs [hunk 3]
```diff
@@ -266,11 +266,11 @@ pub enum ActionError {
...
     PathGroupMismatch(std::path::PathBuf, u32, u32),
-    #[error("`{0}` exists with a different mode ({1:o}) than planned ({2:o}), consider removing it with `rm {0}`")]
+    #[error("`{0}` exists with a different mode ({1:o}) than planned ({2:o}), consider updating it with `chmod {2} {0}`")]
```
Comment (cole-h) [inline src/action/mod.rs:273]:
```suggestion
    #[error("`{0}` exists with a different mode ({1:o}) than planned ({2:o}), consider updating it with `chmod {2:o} {0}`")]
```

File: src/action/mod.rs (hunk 1 above)
Comment (Hoverbear) [reply src/action/mod.rs:269]: These are UIDs so I don't think they're octal?

File: src/action/mod.rs (hunk 2 above)
Comment (Hoverbear) [reply src/action/mod.rs:271]: These are GIDs so I don't think they're octal?

File: src/action/mod.rs (hunk 3 above)
Comment (Hoverbear) [reply src/action/mod.rs:273]: omg...

File: src/action/mod.rs (hunk 1 above)
Comment (cole-h) [reply src/action/mod.rs:269]: Oops, yeah.

File: src/action/mod.rs (hunk 2 above)
Comment (cole-h) [reply src/action/mod.rs:271]: Who knew, brain pattern matching isn't very robust.

**Review by cole-h — APPROVED** [review]

**Pull request merged**